### PR TITLE
designate: add support for SSL (SOC-10877)

### PR DESCRIPTION
**Pull request by flaviodsr**
This change adds support for configuring designate with SSL enabled.

### CODE REVIEW

**Comment by flaviodsr** [comment]
@nicolasbock yes, I will do that once I have this tested.

**Comment by flaviodsr** [comment]
Tested in: https://ci.suse.de/blue/organizations/jenkins/openstack-crowbar/detail/openstack-crowbar/576/pipeline/

File: chef/cookbooks/horizon/recipes/server.rb [hunk 1]
```diff
@@ -432,7 +433,7 @@
     || trove_insecure \
     || sahara_insecure \
     || manila_insecure \
-    || ceilometer_insecure,
```
Comment (nicolasbock) [inline chef/cookbooks/horizon/recipes/server.rb]: Why are we dropping ceilometer here?

Comment (flaviodsr) [reply chef/cookbooks/horizon/recipes/server.rb]: `ceilometer_insecure` is on that list two times. I am removing one of them.

Comment (nicolasbock) [reply chef/cookbooks/horizon/recipes/server.rb]: Ah thanks, I didn't notice.

**Pull request merged**